tests/functional/firefox/test_iostestflight.py

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Tiny nit: missing underscore between `ios` and `testflight` in the filename here `test_iostestflight.py`